test/single_file/unary.expected

```diff
@@ -0,0 +1,119 @@
+}
+
+static int& __dredd_replace_unary_operator_PreDec_int(std::function<int&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ++arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return arg();
+  return --arg();
+}
+
+static int __dredd_replace_unary_operator_PostInc_int(std::function<int&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ++arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return --arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 2)) return arg()--;
+  if (__dredd_enabled_mutation(local_mutation_id + 3)) return ~arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 4)) return -arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 5)) return !arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 6)) return arg();
+  return arg()++;
+}
+
+static int __dredd_replace_unary_operator_PostDec_int(std::function<int&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ++arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return arg()++;
+  if (__dredd_enabled_mutation(local_mutation_id + 2)) return --arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 3)) return ~arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 4)) return -arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 5)) return !arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 6)) return arg();
+  return arg()--;
+}
+
+static int __dredd_replace_unary_operator_Minus_int(std::function<int()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ~arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return !arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 2)) return arg();
+  return -arg();
+}
+
+static float& __dredd_replace_unary_operator_PreInc_float(std::function<float&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return --arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return arg();
+  return ++arg();
+}
+
+static float& __dredd_replace_unary_operator_PreDec_float(std::function<float&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ++arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return arg();
+  return --arg();
+}
+
+static float __dredd_replace_unary_operator_PostInc_float(std::function<float&()> arg, int local_mutation_id) {
+  if (__dredd_enabled_mutation(local_mutation_id + 0)) return ++arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 1)) return --arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 2)) return arg()--;
+  if (__dredd_enabled_mutation(local_mutation_id + 3)) return -arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 4)) return !arg();
+  if (__dredd_enabled_mutation(local_mutation_id + 5)) return arg();
+  return arg()++;
+}
```

Comment:
I think we had agreed that we would not allow replacing post operators with pre operators.

Reply:
I forgot to change this, but it's now fixed.